app/views/dashboard/show.html.erb

```diff
@@ -71,8 +67,11 @@
   </table>
   <br>
   <div class="row">
-    <div class="col-sm-6 col-md-4">
+    <div class="col-sm-12 dashboard-table-header">
       <h1>Case Contacts</h1>
+      <% if policy(:dashboard).create_case_contacts? %>
+        <%= link_to "New Case Contact", new_case_contact_path, class: "btn btn-primary" %>
```

Comment:
Oh I meant to bring this up separately, but we completely removed the "New Case Contact" button for both admins and volunteers in a previous PR. Is that intended, or should we only hide the "New Case Contact" button on this page for admins and keep it visible for volunteers?

Reply:
Hide for admins, keep for volunteers 😅

Reply:
Ok great, that's what I was hoping. The policy check  in the code above should be what we want then!